scheduler.rb

```diff
@@ -15,6 +15,10 @@ module Clockwork
     SendCategoryOverdueEmailsJob.perform_now
   end
 
+  every(1.day, "Send Progress Report Updated Emails", at: "00:00", tz: Rails.application.config.time_zone) do
```

Comment:
this will send whatever updated that day

Reply:
@parndt is this limited to report updates or also trigger when new reports are created? (which it should)

Reply:
Yup it'll work for any